utils/vaults.json

```diff
@@ -933,7 +933,31 @@
     "WANT_ADDR": "0x04068DA6C83AFCFA0e13ba15A6696662335D5B75",
     "WANT_SYMBOL": "USDC",
     "COINGECKO_SYMBOL": "usd-coin",
-    "VAULT_STATUS": "new",
+    "VAULT_STATUS": "active",
     "CHAIN_ID": 250
+  },
+  "mimosa": {
+    "TITLE": "MIMosa",
+    "LOGO": "🤡🔮",
+    "VAULT_ABI": "yVaultV2",
+    "VAULT_TYPE": "experimental",
+    "VAULT_ADDR": "0x2807387F1212cB2852aB97cdAa3627dACa5E4661",
+    "WANT_ADDR": "0x99D8a9C45b2ecA8864373A26D1459e3Dff1e17F3",
+    "WANT_SYMBOL": "MIM",
+    "COINGECKO_SYMBOL": "magic-internet-money",
+    "VAULT_STATUS": "active",
```

Comment:
Silly thing, do you want to put this as `new`? So it will show as new in the website :)